tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -143,7 +131,8 @@ func (c *cluster) Reset() error {
 	for i, m := range c.Members {
 		eps[i] = m.Endpoint
 	}
-	return c.bootstrap(eps)
+	c.agentEndpoints = eps
```

Comment:
Does this need to be updated? I think this code was here because the `agentEndpoints` was taken out of the cluster structure.